src/loaders/HubsTextureLoader.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * @author mrdoob / http://mrdoob.com/
+ */
+
+function loadAsync(loader, url, onProgress) {
+  return new Promise((resolve, reject) => loader.load(url, resolve, onProgress, reject));
+}
+
+function isJPEG(url) {
+  return url.search(/\.jpe?g($|\?)/i) > 0 || url.search(/^data:image\/jpeg/) === 0;
+}
+
+function isPNG(url) {
+  return url.search(/\.png($|\?)/i) > 0 || url.search(/^data:image\/png/) === 0;
+}
```

Comment:
It would be nice if this used `guessContentType` from media-url-utils, into which it would be totally reasonable to put the data-URL logic.